documentation/technical_writing_slides.md

```diff
@@ -0,0 +1,452 @@
+- An **expert** need this material as well – nobody's memory is perfect.
+
+---
+
+## Style Guides (1/4)
+
+Consistent tone and style ...
+
+- makes content easier to read.
+- reduces the reader's cognitive load.
+- increases confidence in authority of content.
+
+Read more: [Write the Docs on style guides](https://www.writethedocs.org/guide/writing/style-guides/)
+
+---
+
+## Style Guides (2/4)
+
+- Different people write differently. Even the same person writes differently every day.
+- Style guide contains a set of standards.
+- Writer does not have to worry -> saves time.
+- Traditional style guides have been around as long as publishing, e.g. [The Chicago Manual of Style](https://www.chicagomanualofstyle.org/).
+
+---
+
+## Style Guides (3/4)
+
+Example: brief excerpt of [Google developer documentation style guide](https://developers.google.com/style)
+
+- **Timeless documentation**: Avoid words such as *"currently"*, *"in the future"*, *"soon"*
+- Use present tense, avoid *"will"*
+    - Bad: *Send a query to the service. The server will send an acknowledgment.*
+
+- **Exclamation points**: Don't use them. Rather use notices such as Note or Caution.
+
+---
+
+## Style Guides (4/4)
+
+- **Headings**: Use "Sentence case of Headings", not "Title Case for Headings"
+- **Image URLs**: Use site-root-relative URL from same domain
+- **Referring to filenames**: code font, word "file" after filename, exact spelling
+    - Good: *In the following `build.sh` file, modify the default values for all parameters.*
+
+- ... even [example person names](https://developers.google.com/style/examples#example-person-names)
+
+---
+
+## Tone of Voice (1/2)
+
+... helps a lot to make tone of documentation consistent.
+
+- If your software/product was a person, how would they be?
+- Let it talk like this person.
+
+> Example preCICE:
+> - Three things to define character? Smart, approachable, responsible.
+> - What kind of smartphone does preCICE use? No iPhone.
+> - Does preCICE have a sense of humor? Yes. When does it use it? Only rarely.
+> - ...
+
+---
+
+## Tone of Voice (2/2)
+
+- Define how to talk to users.
+
+> Example preCICE:
+> - Treat users as peers. preCICE is a CS PhD student, user is a Mechanical Engineering PhD student.
+
+---
+
+## 3. Standard Documentation Building Blocks
+
+---
+
+## `README.md`
+
+**Every** code should contain a README file. There is even [readme driven development](https://tom.preston-werner.com/2010/08/23/readme-driven-development.html).
+
+Minimal variant should contain at least (following [Write the Docs](https://www.writethedocs.org/guide/writing/beginners-guide-to-docs/#readme)):
+
+- What problem the project solves
+- A small code example (for libraries) or how to run (for programs)
+- How to get support and a link to issue tracker
+- Information for people who want to contribute back (also `CONTRIBUTING.md`)
+- Installation instructions
+- The code's license (also `LICENSE`)
+- Frequently Asked Questions (FAQ)
+
```

Comment:
I also read somewhere (not sure what blog) that software this some GUI/visual part should have a screenshot as well.